azure-pipelines.yml

```diff
@@ -31,6 +31,10 @@ steps:
 - bash: |
     npm install --no-save
     npm install -g vsce
+    # Configure git otherwise `npm version` will fail
+    git config --global user.email "adobuildagent@notanemail.com"
+    git config --global user.name "adobuildagent"
+    npm version 1.4.$(Build.BuildId)
```

Comment:
looks good, would be awesome if the 4 would come from a variable as well but we can improve that later